README.md

```diff
@@ -10,7 +10,7 @@ Simply copy and paste a URL into the provided area. Then click shorten URL! Your
 
 ### About:
 
-I created this repo a long time ago (more than 3 years), made this public for Hacktoberfest! This is a very good opportunity for beginners to start their journey with open source. All PRs are welcome!
+I've created this repository over 3 years ago, and have made it public for Hacktoberfest! </br> This is a great opportunity for beginners to start their journey with contributing to open source.</br>All PRs are welcome! :)
```

Comment:
Using trailing whitespace or leaving a line to signify a line-break is cleaner than in-line `<br>` tags.